tests/features/targets/import_targets.feature

```diff
@@ -0,0 +1,19 @@
+Feature: Import targets directly to RSTUF SQL Database
+    As an admin,
+    Admin has deployed and bootstrapped RSTUF
+    Admin has an existent large number of targets (i.e.: 1,000,000 targets)
+    Admin wants to load targets in chunks to process them faster
+
+    Background:
+        Given admin has the RSTUF CLI in a host with access to RSTUF SQL Database
+        And admin has logged in with RSTUF CLI
+        And admin has the credentials to access RSTUF SQL Database
```

Comment:
Do we need some special permissions to access the RSTUF SQL database?
When we add targets the traditional way we require a token with `write:targets` permissions.
I know we skip the API, but how do we flag that a given user with certain credentials is allowed to have access to the database?

Reply:
So, if I understand this correctly once the Warehouse maintainers migrate their targets' data through the CLI skipping the API, and a second time when they are finished they have to make a REST API call to the new endpoint `publish targets` to have them publicly available, right?

Also, I am not sure we answered my concern about access to the database. Who is allowed to skip the API and send targets directly to the SQL database?
Do we want to have a scope scenically for that?
I am trying to think of a name, but no good name comes :D

Reply:
> So, if I understand this correctly once the Warehouse maintainers migrate their targets' data through the CLI skipping the API, and a second time when they are finished they have to make a REST API call to the new endpoint `publish targets` to have them publicly available, right?

No, the CLI does it as one process but in two steps:
1) Import all CSV files passed to the command line and insert them into the RSTUF DB
2) CLI calls the `publish_targets`

It also allows the user to import all CSVs or how many CSV they want to import (and also publish) per command line call.

> Also, I am not sure we answered my concern about access to the database. Who is allowed to skip the API and send targets directly to the SQL database? Do we want to have a scope scenically for that? I am trying to think of a name, but no good name comes :D

Only the administrator that installs the CLI in a host can connect to RSTUF DB and have the credentials.

Reply:
> No, the CLI does it as one process but in two steps:
> 1. Import all CSV files passed to the command line and insert them into the RSTUF DB
> 2. CLI calls the publish_targets
It also allows the user to import all CSVs or how many CSV they want to import (and also publish) per command line call.

Yes, I get it now. The administrator will call "publish_targets" directly to the worker when he is ready.

Reply:
I think it will be a good idea to add one more step when the user uses the CLI to actually publish the targets.

Reply:
I would do both. But let's simplify the user experience.

a) Let's simplify the import process by being a unique command that does 1 and 2. We give a parameter `--skip-publish-targets` for those that want to do it in separate 2 steps. Otherwise, we need to document the process in detail about running commands 'x' and 'y'. It will generate confusion, probable uses saying that they imported but the targets are not in the TUF metadata, because they forgot to run the second command `publish-targets`.

b) We add an admin command to run 'publish-target'. It can be used in other situations.
    It is a single feature in the CLI that will call the endpoint planned in https://github.com/vmware/repository-service-tuf-api/issues/242

Reply:
Based on my last comment, I will add the feature test scenario when the user uses `--skip-publish-targets`